src/globals/formatting.ts

```diff
@@ -1,5 +1,7 @@
 export const formatNumber = function (value: number): string {
-  if (value < 9999) {
+  if (value < 0) {
```

Comment:
this is no longer formatting , suggest to move this out of the function ( or update the function name )

Reply:
debatable if it's no longer formatting.